tools/topology/Makefile.am

```diff
@@ -44,7 +44,8 @@ MACHINES = \
 	sof-apl-dmic-2ch.tplg \
 	sof-apl-dmic-4ch.tplg \
 	sof-icl-dmic-4ch.tplg \
-	sof-apl-src-pcm512x.tplg
+	sof-apl-src-pcm512x.tplg \
+	sof-cnl-rt5682.tplg
```

Comment:
@zhuyingjiang can we please just call it sof-cml-rt5682? It is intended for the cml product and this will make it clear.

Reply:
otherwise the PR looks good

Reply:
@ranj063 It is OK to rename that, my first intention is to indicate this can also be used on CNL/WHL